src/components/header/header.module.css

```diff
@@ -1,12 +1,134 @@
+@import "../../styles/wrapper.css";
+
 .header {
-  font-size: 34px;
-  text-align: center;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 70px;
+  color: #FFF;
+  background: #111;
   margin-bottom: 30px;
+  overflow: hidden;
+}
+
+.headerConteiner {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  width: 40px;
+}
+
+.themes {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  gap: 9px;
+}
+
+.themes li {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  gap: 50px;
+  font-size: 16px;
+  font-weight: 700;
+}
+
+.bascet {
+  width: 24px;
+  height: 24px;
+}
+
+.authButtons {
+  display: none;
+  justify-content: center;
+  align-items: center;
+  gap: 10px;
+}
+
+.authButtons button {
```

Comment:
why do you use this `.authButtons button `? 
isn't enough class `.authButtons`?

Reply:
The .authButtons selector is a group of buttons, and the .authButtons button selector is a specifically button